types/eval-estree-expression/index.d.ts

```diff
@@ -0,0 +1,4 @@
+export function evaluate(tree: any, context: any, options: any): any;
```

Comment:
Is `any` really the best type for these?

In this particular function's case, are you sure it doesn't return at least a `Promise`?